.github/workflows/ci.yml

```diff
@@ -528,6 +528,7 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       have_azure_secrets: ${{ steps.check-secrets.outputs.have_azure_secrets }}
+      have_databricks_secrets: ${{ steps.check-databricks-secrets.outputs.have_databricks_secrets }}
```

Comment:
the cmt title

`Execute Delta Lake Databricks product tests only if corresponding sec…
…rets are provided`

sounds like you're disabling something, whereas you're adding something (that happens to be enabled conditionally)

change commit title to something like

```
Test Delta Lake connector against Databricks runtime
```